hail/www/navbar.css

```diff
@@ -1,87 +1,126 @@
-#hail-navbar  {
-  background-color: #ffffff ;
-/*  min-height: 50px;
-  max-height: 50px;*/
-/*  margin-bottom: 10px;*/
-  border: 0px;
-  border-radius: 0px;
-  border-bottom: 1px solid #e7e7e7;
+#hail-navbar {
+  padding: 0px;
+  margin: 0px;
+  background-color: #fff;
   height: 50px;
-  font-size: 12px;
-  font-family: Helvetica;
-  font-weight: 600;
-  font-color: #555555;
-  margin-bottom: 0px;
+  border-bottom: 1px solid #e7e7e7;
 }
 
-#hail-navbar #hail-navbar-collapse {
-    padding-right: 0px;
-    padding-left: 0px;
+#hail-navbar * {
+  font-size: 0.75rem;
 }
 
 #hail-navbar #hail-navbar-header {
-  box-sizing: border-box;
-  line-height: 20px;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  flex: 1;
+  width: 100%;
+  height: 100%;
 }
 
 #hail-navbar #hail-container-fluid {
-    max-width: 42em;
-    margin: auto;
+  margin: auto;
+  height: 100%;
 }
 
-#hail-navbar #hail-navbar-collapse .nav-item a {
-  padding-top: 28px;
-  padding-bottom: 0px;
-  background-color: #FFFFFF;
+#hail-navbar #hail-menu {
+  margin-top: 0px;
+  flex-direction: row;
+  justify-content: flex-end;
+}
+
+#hail-navbar #hail-havbar-collapse {
+  flex: 1;
+}
+
+#hail-navbar #hail-navbar-collapse {
+  flex-direction: row;
+  flex: 1;
+  border-color: #e7e7e7;
 }
 
 #hail-navbar #hail-navbar-collapse a {
   color: #555555;
   text-decoration: none;
+  outline: none;
+  background: inherit; /*defeat bootstrap focus background*/
 }
 
-#hail-navbar #hail-navbar-collapse .navbar-right {
-  margin-right: -50px;
-}
-
-#hail-navbar .dropdown-menu {
-  min-width: 100px;
-  background-color: #FFFFFF;
-  z-index: 10000;
+#hail-navbar #hail-navbar-collapse a:hover {
+  color: #2a388f;
 }
 
-#hail-navbar #hail-navbar-collapse .nav-item .dropdown-menu a {
-    padding-top: 5px;
-    padding-bottom: 5px;
+#hail-navbar #hail-menu .nav-item {
+  padding: 0px 0.875rem 0px 0.875rem;
+  line-height: 1;
 }
 
-#hail-navbar #hail-navbar-collapse a:hover {
-    color: #2a388f;
+#hail-navbar #hail-menu .nav-item:last-child {
+  padding-right: 0px;
 }
 
 #hail-navbar #hail-navbar-collapse .active.nav-item:not(:hover) {
-    border-bottom: 2px solid #555555;
+  border-bottom: 2px solid #555555;
 }
 
 #hail-navbar #hail-navbar-collapse .active.nav-item:hover {
-    border-bottom: 2px solid #2a388f;
+  border-bottom: 2px solid #2a388f;
 }
 
 #hail-navbar #hail-navbar-collapse .dropdown-item.active {
-    text-decoration: underline;
+  text-decoration: underline;
 }
 
 #hail-navbar #hail-navbar-collapse .dropdown-menu {
   background-color: #ffffff;
 }
 
 #hail-navbar #hail-navbar-collapse {
-   -webkit-font-smoothing: antialiased;
+  -webkit-font-smoothing: antialiased;
```

Comment:
With the font-weight removed (previously line 12), this antialiasing is noticeable on Safari on Mac OS X and is subtly different from the body text (which isn't antialiased). Can we remove this rule entirely?

In particular, this disables sub pixel antialiasing on retina displays, which is, in my subject experience, much nicer to read than standard antialiasing.

Reply:
Could you show me what you're seeing? auto, sub-pixel antialiased, and antialiased give me the same results in safari 12.1.1.

Reply:
It's fixed now, but if you use the dev tools to enable and disable `-webkit-font-smoothing: antialiased` on the navbar, you'll notice that with `antialiased` the weight appears slightly heavier. With no rules, I suspect `subpixel-antialiased` is default enabled and the weight appears normal. I'm probably hyper sensitive, but it looks off when the weight is just a bit heavier than it ought to be.

Reply:
My eyes must not be great.

On body:

Grey:
<img width="1556" alt="Screenshot 2019-08-20 16 12 15" src="https://user-images.githubusercontent.com/5543229/63380862-67acc580-c365-11e9-925c-70a6d54106fe.png">

Sub:
<img width="1594" alt="Screenshot 2019-08-20 16 12 20" src="https://user-images.githubusercontent.com/5543229/63380856-64193e80-c365-11e9-9e82-8c21b7374785.png">

None:
<img width="1591" alt="Screenshot 2019-08-20 16 12 26" src="https://user-images.githubusercontent.com/5543229/63380850-61b6e480-c365-11e9-8425-1265d1a7269d.png">

Similar on header. 

I believe you, I just don't see it. From reading, it may be different with white text on black back